FirebaseUI.podspec

```diff
@@ -1,11 +1,12 @@
 Pod::Spec.new do |s|
   s.name         = 'FirebaseUI'
-  s.version      = '0.5.0'
+  s.version      = '0.5.1'
   s.summary      = 'UI binding libraries for Firebase.'
   s.homepage     = 'https://github.com/firebase/FirebaseUI-iOS'
   s.license      = { :type => 'Apache 2.0' }
   s.author       = 'Firebase'
-  s.source       = { :http => 'https://dl.google.com/dl/firebase/firebaseui/ios/0_5_0/FirebaseUIFrameworks.zip' }
+  # s.source       = { :http => 'https://dl.google.com/dl/firebase/firebaseui/ios/0_5_0/FirebaseUIFrameworks.zip' }
+  s.source       = { :http => 'http://localhost:7777/FirebaseUIFrameworks.zip' }
```

Comment:
Also, you should talk to @yramanchuk about his changes here for local development.